JudoPayDotNet/Clients/ThreeDs.cs

```diff
@@ -13,27 +13,69 @@ internal class ThreeDs : JudoPayClient, IThreeDs
 
         private const string CompleteThreeDAuthorizationAddress = "transactions";
 
+        
+        private const string ResumeThreeDSecureTwoAddress = "resume3ds";
+
+        private const string CompleteThreeDSecureTwoAddress = "complete3ds";
+
+
         private readonly IValidator<ThreeDResultModel> _threeDResultValidator = new InlineValidator<ThreeDResultModel>();
 
+        private readonly IValidator<ResumeThreeDSecureModel> _resumeThreeDSecureValidator = new InlineValidator<ResumeThreeDSecureModel>();
+
+        private readonly IValidator<CompleteThreeDSecureModel> _completeThreeDSecureValidator = new InlineValidator<CompleteThreeDSecureModel>();
+
         public ThreeDs(ILog logger, IClient client)
             : base(logger, client)
         {
         }
 
         public Task<IResult<PaymentRequiresThreeDSecureModel>> GetThreeDAuthorization(string md)
```

Comment:
The transactions/threedauthorisations endpoint is no longer present in PartnerApi so this method can be removed